lib/components/steps/mapping/MappingTab.tsx

```diff
@@ -91,6 +100,12 @@ interface MappingProps {
     defaultCollection: string;
 }
 
+interface CurrentFilterInterface {
```

Comment:
Issue (-blocking): I believe we could reuse the SortingStrategy interface. The sortState is no longer needed with the use of a state variable for the sortOrder